phil/cnx-routes.md

```diff
@@ -0,0 +1,302 @@
+# CNX APIs
+
+https://cnx.org APIs are split into 4 areas and documented in greater detail in each repository.
+
+You can retrieve published content using the [^cnx-archive] APIs (including search), and can change content using the [^cnx-publishing-api] APIs.
+
+Browser URLs are managed and documented in [^webview].
```

Comment:
Note that webview is an in-browser app, and will rewrite many urls before calling to backend servers